test/AutoRest.TestServer.Tests/Mgmt/TestProjects/TestProjectTests.cs

```diff
@@ -291,6 +291,28 @@ public void ValidateSubscriptionExtensionsListResourceByName()
             }
         }
 
+        [Test]
+        public void ValidateParentResourceOperation()
+        {
+            foreach (var operation in FindAllOperations())
+            {
+                var operationObj = Activator.CreateInstance(operation, true);
```

Comment:
I don't think you need to construct one, you can simply inspect the Type itself right?

Reply:
Done